docs/user/04-metrics.md

```diff
@@ -322,6 +322,29 @@ For metrics ingestion to start automatically, simply apply the following annotat
 | `prometheus.io/path`               | `/metrics`, `/custom_metrics` | `/metrics` | Defines the HTTP path where Prometheus can find metrics data.                                                                                                                                                                                                                                                                               |
 | `prometheus.io/scheme`             | `http`, `https` | If Istio is active, `https` is supported; otherwise, only `http` is available. The default scheme is `http` unless an Istio sidecar is present, denoted by the label `security.istio.io/tlsMode=istio`, in which case `https` becomes the default. | Determines the protocol used for scraping metrics — either HTTPS with mTLS or plain HTTP. |
 
+An example configuration for a `Service` might be:
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/port: "8080"
+    prometheus.io/scrape: "true"
+  name: sample
+spec:
+  ports:
+  - name: http-metrics
+    appProtocol: http
+    port: 8080
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: sample
+  type: ClusterIP
+```
+> [!NOTE]
```

Comment:
I'm not a fan of two Note boxes right after each other.
In some cases, both touch the same topics and can be combined in one Note.
In some cases, one (or both) can be formatted as normal text.

Notes should be the exception within normal text (so that they stand out), and shouldn't contain the majority of content.

Reply:
I agree and didn't know better, what is your concrete propsal?

Reply:
Saw you next comment later. Added the note as text above the example